Automattic/jetpack
# Change enabled status flag

Use backend value to set the tier plans enabled flag

## Proposed changes:
This PR adds the `tier-plans-enabled` prop to the ai-assistant payload to make it available for the frontend store. This way, the whole setup can be set from a single source.

The PR also removes the old beta flag that was introduced for the same purpose.

### Other information:


## Jetpack product discussion
p1700679681199019-slack-C054LN8RNVA

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Change the value of the filter, then load the editor and confirm the value is reflected on the store (use redux dev tools):
<img width="471" alt="image" src="https://github.com/Automattic/jetpack/assets/157240/89124d38-6ae9-4500-8a8a-5a52c4cf60d0">

Use the console to switch its value:
```js
wp.data.dispatch( 'wordpress-com/plans' ).setTierPlansEnabled( true|false );
```
